packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -311,11 +328,16 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
       }
 
       await registerAddIn(manifestPath);
-
       // for Outlook, open Outlook.exe; for other Office apps, open the document
-      sideloadFile = (app === OfficeApp.Outlook)
-        ? await getOutlookExePath()
-        : await generateSideloadFile(app, manifest, document);
+      if (app == OfficeApp.Outlook) {
+        const version = await getOutlookVersion();
+        if (version && semver.lt(version, "16.0.13709")) {
+          throw new ExpectedError(`Outlook install version should be 16.0.13709 or greater. Current version: ${version}`);
```

Comment:
This error message would be better if it mentioned that sideload is not supported.

`The current version of Outlook (${version}) does not support sideload. Please use version 16.0.13709 or greater.`